sled-agent/src/services.rs

```diff
@@ -157,7 +170,8 @@ impl ServiceManager {
 
             for addr in &service.addresses {
                 info!(self.log, "Ensuring address {} exists", addr.to_string());
-                let addr_request = AddressRequest::new_static(addr.ip(), None);
+                let addr_request =
```

Comment:
I think a breadcrumb in the vicinity of this code that points to the `create_plan` function in sled-agent/src/rack_setup/service.rs would be helpful here. When reading this code the immediate questions that come to mind are, where are these service specifications coming from and how are these addresses getting decided. I had the benefit of having just seen that code in the review, but for others that may find themselves here trying to sort out where this information came from, could be very helpful.